dependabot/resources/module_list.json

```diff
@@ -91,6 +91,10 @@
             "version_key": "stdlibJdbcVersion",
             "push_to_central": false
         },
+        {
+            "name": "module-ballerinax-java.jms",
+            "push_to_central": false
```

Comment:
Need to add the version key here.

Reply:
Fixed with [94d456aa37d01362551755ca82fb6f5a664f5536](https://github.com/ballerina-platform/ballerina-release/pull/2518/commits/94d456aa37d01362551755ca82fb6f5a664f5536)